wrtbwmon

```diff
@@ -18,6 +18,14 @@ networkFuncs=/lib/functions/network.sh
 uci=`which uci 2>/dev/null`
 nslookup=`which nslookup 2>/dev/null`
 nvram=`which nvram 2>/dev/null`
+ip=/usr/sbin/ip
+[ -x $ip ] || ip=`which ip`
+
+# we need the full version of ip, not busybox
```

Comment:
Is this just for the `ip -6` command below? Busybox and iproute2 both accept the following:
`ip -f inet6 neigh show`